src/types.js

```diff
@@ -3,7 +3,7 @@ type zeroThroughEleven = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11
 
 export type Offset = zeroThroughEleven | void
 
-export type Size = zeroThroughEleven | 'fit' | 'auto' | void
+export type Size = zeroThroughEleven | 12 | 'fit' | 'auto' | void
```

Comment:
Do we need `zeroToEleven` for anything? if not, let's consolidate all into a single type (`zeroToTwelve`).